app/Http/Controllers/Units/SectionsController.php

```
@@ -0,0 +1,110 @@
+            ->sections()
+            ->orderBy('ord', 'asc')
+            ->orderBy('created_at', 'asc')
+            ->with('slots', 'unit:id,slug')
+            ->paginate(15);
+
+        return Inertia::render('units/structure/sections/list', [
+            'sections' => $sections,
+        ]);
+    }
+
+    public function show(Unit $unit, Section $section)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$unit'. (undefined)

(UnusedFormalParameter)